frappe/desk/query_report.py

```diff
@@ -464,12 +474,12 @@ def add_total_row(result, columns, meta=None):
 		for row in result:
 			if i >= len(row):
 				continue
-
 			cell = row.get(fieldname) if isinstance(row, dict) else row[i]
 			if fieldtype in ["Currency", "Int", "Float", "Percent", "Duration"] and flt(
 				cell
 			):
-				total_row[i] = flt(total_row[i]) + flt(cell)
+				if not (is_tree and row.get(parent_field)):
```

Comment:
For nested doctypes, we are adding percentages and then dividing by the sum of all nodes (parents as well as leaves)
Is this value mathematically correct?
I think we should just show a '-' when the datatype of the column is a percentage

Reply:
Not possible currently, should be fixed via datatable